app/modules/topics/saga/task/updateWithContent.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import { type Saga } from 'redux-saga';
+import { put, call, select } from 'redux-saga/effects';
+
+import { ObjectNotFoundError } from 'errors';
+import asyncRequests from 'modules/asyncRequests';
+import contentItems from 'modules/contentItems';
+
+import actions from '../../actions';
+import * as a from '../../actionTypes';
+import * as m from '../../model';
+import selectors from '../../selectors';
+
+// eslint-disable-next-line require-yield
+const updateWithContent = function* (action: a.UpdateWithContentAction): Saga<void> {
+  const { id, title, description } = action.payload;
+  const topic: ?m.Topic = yield select(selectors.getById, { id });
+  if (topic == null) throw new ObjectNotFoundError(`topics:topic`, id);
+
+  if (title !== null && description !== null) {
```

Comment:
Note: a `null` check here will probably not work the way you intended it, since the `validate.stringProps()` function call currently happens in `apiPatch` instead of in `updateWithContent`, so empty strings have not yet thrown an error (for non-nullable props) or been converted to `null` (for nullable props) at this point. Also, `description` is supposed to be nullable, so it being `null` should not prevent the `title` from being patched.

Reply:
Oops, should read something like 
```js
if (!(title === null && description === null))
```

But if I get this right, I can omit this `null` check, call `apiPatch` and modify the validation there to only have optional props.